robottelo/rhsso_utils.py

```diff
@@ -118,3 +120,68 @@ def delete_rhsso_user(username):
     run_command(
         cmd=f"{KEY_CLOAK_CLI} delete -r {realm} users/{user_details['id']}", hostname=rhsso_host,
     )
+
+
+def update_client_configuration(json_content):
+    """Update the client configuration"""
+    client_id = get_rhsso_client_id()
+    upload_rhsso_entity(json_content, "update_client_info")
+    update_cmd = (
+        f"{KEY_CLOAK_CLI} update clients/{client_id} -f update_client_info -s enabled=true --merge"
+    )
+    run_command(cmd=update_cmd, hostname=rhsso_host)
+
+
+def get_oidc_token_endpoint():
```

Comment:
These functions have limited use when they're statically returning a string formatted with data from settings. Consider taking hostname and realm as kwargs (for this function, other kwargs for the other similar functions), and constructing the strings from either the provided kwargs or values from settings.

Reply:
Hence this is dependant of https://github.com/SatelliteQE/robottelo/issues/7990 issue will work on later sprint